src/UartSerial.cpp

```diff
@@ -27,6 +23,8 @@ UartSerial::~UartSerial()
 
 size_t UartSerial::available()
 {
+  if (_fd == -1) return 0;
+
   int numBytesAvailable;
```

Comment:
Hate for the code to look like 7 different people were in it naming things. I'll switch to the lowercase/underscore approach for all variables unless you object.